packages/math-input/src/components/input/math-input.tsx

```diff
@@ -196,9 +198,47 @@ class MathInput extends React.Component<Props, State> {
             }
         };
 
+        // We want to allow the user to blur the input by clicking outside of it
+        // when using ChromeOS third-party browsers that use mobile user agents,
+        // but don't actually simulate touch events.
+        this.blurOnClickOutside = (evt: any) => {
+            if (this.state.focused) {
+                if (!this._container.contains(evt.target)) {
+                    if (
+                        this.props.keypadElement &&
+                        this.props.keypadElement.getDOMNode()
+                    ) {
+                        let touchDidStartInOrBelowKeypad = false;
+                        const bounds = this._getKeypadBounds();
+
+                        const [x, y] = [evt.clientX, evt.clientY];
+
+                        // We only want to blur if the click is above the keypad,
+                        // to the left of the keypad, or to the right of the keypad.
+                        // The reasoning for not blurring for any clicks below the keypad is
+                        // that the keypad may be anchored above the 'Check answer' bottom bar,
+                        // in which case we don't want to dismiss the keypad on check.
+                        if (
+                            (bounds.left <= x &&
+                                bounds.right >= x &&
+                                bounds.top <= y &&
+                                bounds.bottom >= y) ||
+                            bounds.bottom < y
+                        ) {
+                            touchDidStartInOrBelowKeypad = true;
```

Comment:
Absolutely, I'll get that set up! I was a little worried it would make the logic fairly confusing to read, but I think a better solution is to make a "isWithinKeypadBounds" function that can return a boolean in order to clean up the if statement. That should make things far more readable and a little more dry :)